config.default.yaml

```diff
@@ -340,18 +340,38 @@ costs:
 
 
 monte_carlo:
+    # uniform: https://chaospy.readthedocs.io/en/master/api/chaospy.Uniform.html
```

Comment:
This description is unfortunately not clear.

I drop here few comments to potentially improve:
- We could add a first line describing the option list as a short introduction. we should mention that a different distribution can be specified to any different pypsa network parameter and the supported distributions are the following
- I think we can avoid a link for every option, we could reference the chaospy reference guide and mention the supported distributions. 
- the details on how to specify the parameters for each distribution may be moved below under "uncertainties". There it is worth adding few comments to specify how to specify the parameters. now it is kind-of everything at the beginning and it may be a bit complex

Reply:
Thanks for the review. I have made changes to the descriptions.